assets/src/adaptivity/scripting.ts

```diff
@@ -222,6 +222,9 @@ export const getExpressionStringForValue = (
     // val = convertExponentialToDecimal(val);
     val = parseFloat(val);
     if (val === '') {
+      // TODO: Figure out the intent here.
```

Comment:
This is why the script sets the value to NaN, the non-functional code would make me think we should set it to null instead, but left it as-is for now.